pkg/kube/snapshot/snapshot_beta.go

```diff
@@ -411,6 +411,18 @@ func UnstructuredVolumeSnapshotClass(gvr schema.GroupVersionResource, name, driv
 			},
 			VolSnapClassBetaDriverKey: driver,
 			"deletionPolicy":          deletionPolicy,
+			"parameters":              Mss2msi(params),
 		},
 	}
 }
+
+func Mss2msi(in map[string]string) map[string]interface{} {
```

Comment:
Can you please add comment to the function? It will help understand its purpose.

Reply:
Added a comment to clarify